### ISSUE TITLE
Possible mixup between API and SDK? 

### ISSUE THREAD

**Issue by utezduyar** [open-telemetry/opentelemetry-specification#3007]
SAMPLING: https://opentelemetry.io/docs/reference/specification/trace/sdk/
`The flag combination SampledFlag == true and IsRecording == false could cause gaps in the distributed trace, and because of this OpenTelemetry API MUST NOT allow this combination.`

I believe it is "OpenTelemetry **SKD** MUST NOT allow" not "OpenTelemetry **API** MUST NOT allow". It is the SDK that needs to enforce the recommended implementation.

**Comment by utezduyar** [comment]
@yurishkuro Can you look at this please? If I have a point, I will fix the spec, otherwise we can just close it.

**Comment by yurishkuro** [comment]
Yes, the sentence should say SDK, but the bigger problem is that I think it is incorrect altogether. The combination is not only possible, but required to implement some behaviors. For instance, the instrumentation_scope argument to GetTracer was introduce to allow suppressing data collection at certain layers. To avoid the same problem of gaps, the suppression may work by returning a span with the same span id as the parent, the same sampled=true, but recording=false. The child of such span may continue to be sampled and be recording.
